app/client/ui/FormSuccessPage.ts

```diff
@@ -57,6 +59,11 @@ const cssFormSuccessMessageImageContainer = styled(cssFormMessageImageContainer,
   height: 215px;
 `);
 
+const cssFormSuccessMessageHeading = styled('h1', `
```

Comment:
Can you explain this? I don't get the idea of wrapping image inside h1.

Reply:
Sure. The main idea is that headings are helpful when using a screen reader. Quickly scanning headings on the page is something that help getting the gist of what is on a page (thanks to headings-specific keyboard shortcuts exposed by the screen reader), it's a rather usual workflow.

Visually, the image is the main thing showing us the context at load, with its big green checkbox showing us the submission was a success; it kinda acts as a heading.

So we can just wrap the image with an actual heading and give the image the proper alt text. With a screen reader the alt text will be the thing announced as heading.

This is not that important but it improves things a tiny bit.